kubernetes-typescript/package.json

```diff
@@ -6,7 +6,7 @@
     },
     "dependencies": {
         "@pulumi/pulumi": "^3.0.0",
-        "@pulumi/kubernetes": "^3.0.0",
```

Comment:
After this change kubernetes-typescript template doesn't work because kubernetesx conflicts with kubernetes 4.x https://github.com/pulumi/pulumi-kubernetesx/blob/097ea75f38fc7167894382e431bd0cf5ac7a614d/nodejs/kubernetesx/package.json#L24